src/EnergyPlus/OutputProcessor.hh

```diff
@@ -614,15 +659,15 @@ namespace OutputProcessor {
     inline void ReallocateIVar(EnergyPlusData &state);
 
     TimeStepType ValidateTimeStepType(EnergyPlusData &state,
-                                      std::string const &TimeStepTypeKey, // Index type (Zone, HVAC) for variables
-                                      std::string const &CalledFrom       // Routine called from (for error messages)
+                                      eTimeStepType const &TimeStepTypeKey, // Index type (Zone, HVAC) for variables
```

Comment:
ValidateTimeStepType also takes the enum, but I don't know if this is necessary anymore...need to check.

Reply:
Now I recall, this is still necessary because there is another optimization to be done on this enumeration.  In develop, there are 6 different strings you can pass in to SetupOutputVariable for the time step type.  I created an enumeration to match those 6.  There is also a deeper enumeration that takes those 6 and splits them into two groups.  I should make another pass to just eliminate the list of 6 and pass one of the 2 deeper enumerations directly so that this doesn't need to be validated.  I am inclined to do that on a follow-up branch but I'm open to suggestion.

Reply:
Same exact thing with variable type, and note that the outcome of this will ultimately fix the weird enum naming, so that comment is fairly moot now.

Reply:
Yes, the new enums aren't needed, just use the existing ones everywhere, now or in a second pass. 
```
    enum class StoreType
    {
        Averaged = 1, // Type value for "averaged" variables
        Summed        // Type value for "summed" variables
    };

    enum class TimeStepType
    {
        TimeStepZone = 1,   // Type value for "zone" timestep variables
        TimeStepSystem = 2, // Type value for "system" timestep variables
    };
```

Reply:
I wonder...do those numeric values end up having meaning in SQL?